pages/outreachy.md

```diff
@@ -54,8 +57,56 @@ Interested in becoming a mentor? Contact one of the coordinators!
 
 ## Available tasks
 
-Coming soon. At this moment, we recommend looking at the following issue trackers for smaller tasks that might
-give a sense of what's coming:
+### Instrumentation for mobile applications
+
+Mentor: Gary Brown
+
+* Collect ideas on what's possible and what's desirable for mobile applications
+* Decide and experiment with different options for an optimized mobile experience, like, which sampling strategies
+make more sense
+* Create the missing pieces, like OpenTracing framework integration for some Android component, or a specific Jaeger
+tracer for mobile usage
+* Documentation + examples
+* The end result would be: a trace starts when a user opens the app on the phone and backend microservices are called
+
+### Split the JavaScript OT library: NodeJS and Browser
+
+Mentor: Pavol Loffay
+
+* HttpSender which works in a web browser
+* Intercept web browser events (maybe can be done as OpenTracing instrumentation)
+
+### Drive consensus on a common format for reporting trace/span data
```

Comment:
I am not sure this is a good intern project, it requires a lot more experience with the tracing space

Reply:
Reason I thought this might be good is that the intern project is just to document and implement prototype (one or more language tracers/jaeger backend adapter) - guidance on what should be in the standard format should come from the community, so their role would be to request input and feed that into their prototype. The initial starting point would be that it must capture the information provided via the OpenTracing API.
@bhs Thoughts?

Reply:
If we keep this, could you send a PR to change this page with the outcome of the discussion?

Reply:
@yurishkuro Would rewording as @bhs suggests be ok, or would you still prefer it is removed?

Reply:
Yes. Can we add some description?